2-maven-build.sh

```diff
@@ -5,11 +5,12 @@ if [ -z "$JENKINS_URL" ]; then
 fi
 
 mvn \
-  -f releng \
+  deploy \
```

Comment:
should we add clean goal here anyway?

Reply:
It would be useless because in the CI the environment is always clean (it wasn't there before either); of course, I guess it would be harmless (it would basically only clean xtend-gen folders). So if you want we can add it.

Reply:
i use the 1 2 3 scripts also locally

Reply:
I added 'clean' to the Maven script